Cubical/HITs/FreeGroup/Base.agda

```diff
@@ -0,0 +1,28 @@
+{-
+
+This file contains:
+
+- An implementation of the free group of a type of generators as a HIT
+
+-}
+{-# OPTIONS --safe #-}
+
+module Cubical.HITs.FreeGroup.Base where
+
+open import Cubical.Foundations.Prelude
+
+private
+  variable
+    ℓ : Level
+
+data FreeGroup (A : Type ℓ) : Type ℓ where
+  η     : A → FreeGroup A
+  m     : FreeGroup A → FreeGroup A → FreeGroup A
```

Comment:
_·_ is a better name

Reply:
I used the book notation on this one, chapter 6.11. I can change it if you think it is necessary but I think it is better to keep it that way.

Reply:
I agree that · is nicer. I don't think it's necessary to follow the HoTT book notations if they are not very nice :-)

Reply:
I changed m to · and also e to ε.